ui/pages/Transaction.tsx

```
@@ -15,21 +15,31 @@ import TextAd from 'ui/shared/ad/TextAd';
 import useTxQuery from 'ui/tx/useTxQuery';
 
+// Lazy load heavy/non-critical tab components for better SSR streaming
+const TxAssetFlows = lazy(() => import('ui/tx/TxAssetFlows'));
+const TxBlobs = lazy(() => import('ui/tx/TxBlobs'));
+const TxInternals = lazy(() => import('ui/tx/TxInternals'));
+const TxLogs = lazy(() => import('ui/tx/TxLogs'));
+const TxRawTrace = lazy(() => import('ui/tx/TxRawTrace'));
+const TxState = lazy(() => import('ui/tx/TxState'));
+
+// Import skeleton components for Suspense fallbacks
+import { TxAssetFlowsSkeleton } from 'ui/tx/skeletons/TxAssetFlowsSkeleton';
```

ESLint

[error] `ui/tx/skeletons/TxAssetFlowsSkeleton` import should occur before import of `ui/tx/TxAuthorizations`

(import-helpers/order-imports)